src/screens/History/MultiChainHistoryEtherspot.js

```diff
@@ -34,27 +33,67 @@ import { accountHistorySelector } from 'selectors/history';
 import { accountAssetsSelector } from 'selectors/assets';
 import { accountCollectiblesHistorySelector } from 'selectors/collectibles';
 
+// Constants
+import { CHAIN } from 'constants/chainConstants';
+
 // Utils
 import { getHistoryEventsFromTransactions, parseHistoryEventFee } from 'utils/history';
 import { addressesEqual, getAssetsAsList } from 'utils/assets';
+import { useChainsConfig } from 'utils/uiConfig';
 
 // Types
 import { EVENT_TYPE, type Event } from 'models/History';
+import type { Chain } from 'models/Chain';
 
-function HistoryListEtherspot() {
-  const items = useHistoryEvents();
+function MultiChainHistoryEtherspot() {
+  const [tabIndex, setTabIndex] = useState(0);
 
-  return <HistoryList items={items} />;
+  const chainsConfig = useChainsConfig();
+
+  const chains = [
+    CHAIN.ETHEREUM,
+    CHAIN.POLYGON,
+    CHAIN.BINANCE,
+    CHAIN.XDAI,
+  ];
+
+  const items = chains.map((chain) => {
+    const { title } = chainsConfig[chain];
+
+    return {
+      key: chain,
+      title,
+      component: () => <ChainHistoryView chain={chain} />,
```

Comment:
It looks that there is a very subtle bug here, if you add following code to HistoryList component:
```  React.useEffect(() => {
    console.log("HISTORY LIST MOUNT", items?.length);
    return () => console.log("HISTORY LIST UNMOUNT", items?.length);
  }, []);
```

you can observe that the components get unmounted & mounted on every render.  This is described in detail here: https://github.com/satya164/react-native-tab-view#renderscene-required

Reply:
Basically TabView component needs a slight update to handle that properly. I can add that to this PR if you want.

Reply:
I assume this is fixed 👍  thanks for help